lib/teslamate_web/live/charge_live/cost.html.leex

```diff
@@ -115,7 +115,7 @@
         </p>
         <p class="control is-expanded">
           <%= text_input f, :cost, class: "input",
-                                   type: :number, inputmode: :decimal, step: 0.01,
+                                   type: :number, inputmode: :decimal, step: 0.0001,
```

Comment:
The `step` does not match the `scale` of 2 defined in the migration below.

Reply:
So the problem here is the HTML field is used as calculation field and also used to store the result.

The calculation requires four decimal places to satisfy the requirement of bug 1540, wheras the result uses 2 decimal places.  For the time being I am going to increase the size of the cost field in the charging_processes table to compensate.